custom_components/variable/helpers.py

```
@@ -44,24 +50,28 @@ def _parse_attribute_path(path: str) -> list:
 
 
 def looks_like_attribute_path(path: str) -> bool:
+    """Return whether an attribute name uses supported nested-path syntax."""
     # Only treat bracket notation as nested paths (e.g., "test[a]")
-    # Treat dot notation as literal attribute names for backward compatibility (e.g., "test.a" stays as-is)
+    # Treat dots as literal attribute names for backward compatibility.
     if "[" in path:
         return True
     return False
```

Ruff (0.16.1)

[warning] Return the condition `"[" in path` directly

Replace with `return "[" in path`

(SIM103)